src/components/TopicTelephoneInfos/TopicTelephoneInfos.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../globals.scss';
+
+.wkp-tel-infos {
+  position: absolute;
+  top: 110px;
+  left: 50%;
+  transform: translate(-50%, 0);
+  z-index: 1;
+  height: 0;
+  background-color: white;
+  border: 0;
```

Comment:
@edellucien why do you set padding and height to "0"?

Reply:
```suggestion
```

Reply:
Indeed it is unecessary, setting it on focus is sufficient